packages/kbn-monaco/src/register_globals.ts

```diff
@@ -46,3 +46,7 @@ window.MonacoEnvironment = {
     return new Worker(URL.createObjectURL(blob));
   },
 };
+
+// needed for functionsl tests so that we can get value from 'editor'
+// @ts-ignore
+window.monaco = monaco;
```

Comment:
Could you please move it into `MonacoEnvironment`.

```
window.MonacoEnvironment = {
   monaco,
   etWorker: (module: string, languageId: string) => {,
   .....
}
```

Reply:
done